sql_user_check_sql_server/database_stats.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+mode="stats"
+port="1433"
+
+
+while getopts h:u:d:o:m:p flag
+do
+    case "${flag}" in
+        h) dbhost=${OPTARG};;
+        u) username=${OPTARG};;
+        d) database=${OPTARG};;
+        o) outputfile=${OPTARG};;
+        m) mode=${OPTARG};;
+        p) port=${OPTARG};;
+    esac
+done
+
+
+echo "dbhost: $dbhost";
+echo "username: $username";
+echo "database: $database";
+echo "mode: $mode";
+echo "outputfile: $outputfile";
+echo "port: $port";
```

Comment:
No need to print these.

If you do want to print it print it in one line with proper message

Reply:
done, printing in one line.
I think printing them is useful so that user can know what values they have entered for flags.